buildSrc/build.gradle

```diff
@@ -118,10 +118,10 @@ dependencies {
   api 'org.apache.maven:maven-model:3.6.2'
   api 'com.networknt:json-schema-validator:1.0.36'
   api "com.fasterxml.jackson.core:jackson-databind:${props.getProperty('jackson_databind')}"
-  api "org.bouncycastle:bcprov-jdk15on:${props.getProperty('bouncycastle')}"
-  api "org.bouncycastle:bcpkix-jdk15on:${props.getProperty('bouncycastle')}"
-  api "org.bouncycastle:bcpg-jdk15on:${props.getProperty('bouncycastle')}"
-  api "org.bouncycastle:bcutil-jdk15on:${props.getProperty('bouncycastle')}"
+  api "org.bouncycastle:bcprov-jdk15to18:${props.getProperty('bouncycastle')}"
```

Comment:
So this is what we get:
 - the `org.bouncycastle:*`  overrides were needed for `api 'com.netflix.nebula:gradle-info-plugin:8.2.0'
 - those are explicitly using  `xxx-jdk15on`

Reply:
This is how the dependencies look like now

Reply:
Ok I will change it back to 1.70 for jdk15on

Reply:
@reta done

Reply:
As we agreed, let's do substitution instead:

```
  modules {
    module("org.bouncycastle:bcprov-jdk15on") {
      replacedBy("org.bouncycastle:bcprov-jdk15to18", "the jdk15on artifacts are not supported anymore")
    }
    module("org.bouncycastle:bcpkix-jdk15on") {
      replacedBy("org.bouncycastle:bcpkix-jdk15to18", "the jdk15on artifacts are not supported anymore")
    }
    module("org.bouncycastle:bcpg-jdk15on") {
      replacedBy("org.bouncycastle:bcpg-jdk15to18", "the jdk15on artifacts are not supported anymore")
    }
    module("org.bouncycastle:bcutil-jdk15on") {
      replacedBy("org.bouncycastle:bcutil-jdk15to18", "the jdk15on artifacts are not supported anymore")
    }
  }
```